Makefile

```diff
@@ -3,17 +3,22 @@ export DJANGO_CONFIGURATION=Test
 export PYTHONWARNINGS=ignore,default:::fragdenstaat_de
 
 test:
-	flake8 fragdenstaat_de
+	ruff check
 	pytest --reuse-db
 
 testci:
 	coverage run --branch -m pytest --reuse-db
 	coverage report
 
-requirements: requirements.in requirements-dev.in requirements-production.in
-	pip-compile requirements.in
-	pip-compile requirements-dev.in
-	pip-compile requirements-production.in
+requirements: pyproject.toml
+	uv pip compile -o requirements.txt pyproject.toml
+	uv pip compile -o requirements-dev.txt --extra dev pyproject.toml
+	uv pip compile -o requirements-production.txt --extra production pyproject.toml
+
+dependencies:
+	./devsetup.sh upgrade_frontend_repos
+
+sync_frontend_deps: requirements dependencies
```

Comment:
Maybe name them `frontend_dependencies`, `backend_dependencies` and `dependencies` so it is clear what does what? `sync_frontend_deps` also refreshing all lockfiles is a bit counterintuitive to me